tsconfig.json

```diff
@@ -8,10 +8,6 @@
         "allowSyntheticDefaultImports": true,
 
         "outDir": "dist",
-
```

Comment:
Curious: What makes this no longer necessary with v11?

Reply:
I think v11 fixed the file paths. I couldn't get this to compile once I upgraded to v11 without removing this alias